packages/components/psammead-consent-banner/src/index.jsx

```diff
@@ -80,35 +80,35 @@ const Options = styled.ul`
   list-style-type: none;
 
   & li + li {
-    padding-top: ${GEL_SPACING};
+    margin-top: 0.75rem;
```

Comment:
Where does this come from on the design? Can that use a GEL constant?

Reply:
0.75rem = 12px, and the gap between the text and the top/bottom of the button is 12px

Reply:
here this appears to align with 3 'spacing units' defined here, maybe name a constant after that? https://www.bbc.co.uk/gel/guidelines/spacing-units I'd consult with the UX guys to see if this is how they made their spacing choices.